src/components/vm/filesystems/vmFilesystemsCard.jsx

```diff
@@ -248,21 +247,9 @@ const VmFilesystemAddModal = ({ connectionName, memory, memoryBacking, objPath,
                     <Grid hasGutter>
                         {!memoryBacking &&
                         <FormGroup hasNoPaddingTop
-                                   helperText={_("Using virtiofs requires setting up shared memory")}
-                                   fieldId={`${idPrefix}-modal-file-backed`}
+                                   fieldId={`${idPrefix}-modal-memfd-backed`}
                                    label={_("Memory backing")}>
-                            <Radio id={`${idPrefix}-modal-file-backed`}
-                                   isChecked={memoryBackingType == "file" }
-                                   label={_("File backed")}
-                                   name="memoryBackingType"
-                                   onChange={(_, event) => setMemoryBackingType(event.currentTarget.value)}
-                                   value="file" />
-                            <Radio id={`${idPrefix}-modal-hugepages`}
-                                   isChecked={memoryBackingType == "hugepages" }
-                                   label={_("Hugepages")}
-                                   name="memoryBackingType"
-                                   onChange={(_, event) => setMemoryBackingType(event.currentTarget.value)}
-                                   value="hugepages" />
+                            <div id={`${idPrefix}-modal-memfd-backed`}>{_("Using virtiofs requires shared memory. memfd memory backing will be automatically configured for this guest")}</div>
```

Comment:
This is quite a technicality, and feels like an internal implementation detail. Does this really need to be exposed in the UI? (this is a long and complicated translatable string, after all)

Reply:
The reason I exposed it - it because there are 3 available memory backing options from libvirt. After discussions with the virt team - I went with this one (memfd). I thought the users might be as well informed about their guess config changing to enable memory backing once they set up a shared filesystem.

In addition - memory backing configuration is added - if not yet present in the guest XML - but it's not removed when removing the shared folders. For this reason - I find it important that the user get's notified about it both on addind and also on removal.

Reply:
As we talked over IRC _ removed the string.